src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,2 @@
+package core.basesyntax;public class FIgureSupplier {
```

Comment:
The class 'FigureSupplier' is empty. It should contain methods and fields that provide functionality as per the task requirements.